domains/postgres/init.go

```diff
@@ -40,6 +40,28 @@ func Migration() (*migrate.MemoryMigrationSource, error) {
 					`DROP TABLE IF EXISTS domains`,
 				},
 			},
+			{
+				Id: "domain_2",
+				Up: []string{
+					`CREATE TABLE IF NOT EXISTS invitations (
+						invited_by      VARCHAR(36) NOT NULL,
+						user_id         VARCHAR(36) NOT NULL,
+						domain_id       VARCHAR(36) NOT NULL,
+						role_id         VARCHAR(36) NOT NULL,
+						role_name       VARCHAR(200),
```

Comment:
Please remove `role_name` from invitations 
We retrieve role name by using roles repo